### PR TITLE
Better work items

### PR DESCRIPTION
**Pull request by samoht**
On top of #44

### CODE REVIEW

**Comment by samoht** [comment]
If think we can get these reports even more structured (to avoid all the side-effects in the `pp` function) but that can come later.

**Comment by samoht** [comment]
(actually it's missing one more bit of code to use `Item` inside `Reports`)

File: src/aggregate.ml [hunk 1]
```diff
@@ -337,16 +298,16 @@ let rec process t ht ast =
   | hd :: tl -> process t ht (process_entry t ht hd tl)
   | [] -> ()
 
-let process ?(ignore_sections = [ "OKR Updates" ]) ?(include_sections = []) ast
-    =
+let of_makdown ?(ignore_sections = [ "OKR Updates" ]) ?(include_sections = [])
```
Comment (MagnusS) [inline src/aggregate.ml:301]:
```suggestion
let of_markdown ?(ignore_sections = [ "OKR Updates" ]) ?(include_sections = [])
```

**Comment by MagnusS** [comment]
LGTM - do you want to merge as is or add more?

**Comment by samoht** [comment]
I've force pushed a new version (with a custom printer). I think this is ready to merge now.

**Review by patricoferris — APPROVED** [review]
This LGTM, I think it captures the subset of markdown that most people use. For example most links are `[tarides](https://tarides.com)`, but you can write `[tarides](https://tarides.com "tarides")` which sets the title... I've never seen this in the wild though :))

**Comment by MagnusS** [comment]
Thanks! Since this changes the report aggregation I'll run some tests on our old reports to see if there's any unintended changes before merging

**Comment by MagnusS** [comment]
Just checked now. The diff varies a lot due to the different day format and other minor changes, but from the items I checked it looked correct - and tests are green. The formatting is also better, so it may fix some issues reported in #38 

Would it be possible to also add a newline after each list of work items? We had this in the old format and it makes it easier to separate KR entries from long lists of work items.

**Comment by MagnusS** [comment]
Thanks! Ci failures look unrelated so merging

**Pull request merged**